packages/dialtone-vue3/components/datepicker/modules/month-year-picker.vue

```diff
@@ -1,73 +1,132 @@
 <template>
-  <div class="d-datepicker__month-year-picker">
-    <div>
-      <button
-        id="prevYearButton"
-        :ref="el => { if (el) setDayRef(el) }"
-        type="button"
-        :aria-label="`${changeToLabel} ${prevYearLabel} ${selectYear - 1}`"
-        @click="changeYear(-1)"
-        @keydown="handleKeyDown($event)"
+  <dt-stack
+    direction="row"
+    class="d-datepicker__month-year"
+    gap="300"
+  >
+    <dt-stack
+      as="nav"
+      direction="row"
+      gap="200"
+      class="d-datepicker__nav"
+    >
+      <dt-tooltip
+        :message="prevYearLabel"
+        placement="top"
       >
-        <dt-icon
-          name="chevrons-left"
-          size="400"
-        />
-      </button>
-      <button
-        id="prevMonthButton"
-        :ref="el => { if (el) setDayRef(el) }"
-        type="button"
-        :aria-label="`${changeToLabel} ${prevMonthLabel} ${formattedMonth(selectMonth - 1, MONTH_FORMAT)}`"
-        @click="changeMonth(-1)"
-        @keydown="handleKeyDown($event)"
+        <template #anchor>
+          <dt-button
+            id="prevYearButton"
+            :ref="el => { if (el) setDayRef(el) }"
+            size="xs"
+            importance="clear"
+            class="d-btn--icon-only d-datepicker__nav-btn d-btn--muted"
```

Comment:
Should be able to do...
```
size="xs"
importance="clear"
kind="muted"
circle
class="d-datepicker__nav-btn"
```

Reply:
👍🏼 
Added 
```
kind="muted"
circle
```
and clered class to `class="d-datepicker__nav-btn"` PR #95 vue2 commit https://github.com/dialpad/dialtone/pull/95/commits/760a0f3eb3862521934b5ffb0d25c91d1c5c023d